src/views/Predictions/Mobile.tsx

```diff
@@ -21,7 +21,7 @@ const StyledMobile = styled.div`
   height: 100%;
   max-height: 100%;
 
-  ${({ theme }) => theme.mediaQueries.lg} {
+  ${({ theme }) => theme.mediaQueries.xl} {
```

Comment:
Can you double check the rest of the media queries in Predictions, they all need to go to `xl`

Reply:
I checked it again and there is no need to change anything else.
@hachiojidev